backend/dataall/modules/shares_base/api/resolvers.py

```diff
@@ -214,7 +213,7 @@ def resolve_user_role(context: Context, source: ShareObject, **kwargs):
 
 
 def resolve_can_view_logs(context: Context, source: ShareObject):
-    return ShareLogsService.check_view_log_permissions(context.username, context.groups, source.shareUri)
+    return ShareLogsService.check_view_logs_permissions(context.username, context.groups, source.shareUri)
```

Comment:
Why have you changed the previous? Right now getting a shareObject is going to result in exception if it is disabled due to this resolver - which means that we cannot read the rest of the info of the ShareObject
```
def resolve_can_view_logs(context: Context, source: ShareObject):
    try:
        return ShareLogsService.check_view_logs_permissions(context.username, context.groups, source.shareUri)
    except Exception as e:
        log.error(f'Failed to resolve if user can view share logs due to: {e}')
        return False
```

Reply:
Updated code with the suggested changes